onebusaway-android/src/main/java/org/onebusaway/android/tripservice/PollerTask.java

```diff
@@ -151,7 +156,6 @@ private long getReminderMin(String tripId, String stopId) {
      * Checks arrivals from given ObaArrivalInfoResponse
      *
      * @param response arrival information
-     * @param tripId
```

Comment:
not sure how `tripId` got deleted-- might have been the code formatter. I'll add it back in with a description 👍